### ISSUE TITLE
Cannot open Jamtaba as AU in Ableton in Mac OS Big Sur

### ISSUE THREAD

**Issue by Feliforever** [elieserdejesus/JamTaba#1407]
I cannot use Jamtaba as AU in Ableton Live running in Mac OS Big Sur. Since I updated the OS I've been having this problem. When I import the plug in to the master track and try to open it, it just opens a black screen. I can use Jamtaba as a stand alone with no problems but not as a plug in. I have the latest version of JT and Ableton Live 10. 
![Captura de pantalla 2021-10-14 a las 20 46 40](https://user-images.githubusercontent.com/92555112/137410249-890b9a89-451f-4931-8125-1043508aded1.png)

**Comment by berndkeul** [comment]
I can run it in Monterey in a M1 Mac mini , but it chrashes Ableton . so I can use it but cannot save anything ...

**Comment by jbaylies** [comment]
I have not had that experience. JamTaba is working well for me in Ableton 11.2.7 on m1 macbook pro 13.2

**Comment by peteone43** [comment]
> I can run it in Monterey in a M1 Mac mini , but it chrashes Ableton . so I can use it but cannot save anything ...


simillar isssue MacOS Ventura 13.2, Ableton 11.2.7 second i put jamtaba plugin on i get a beachball spinning for a minute and then computerfreezes completly i have to power it down... :-((((((((((((

The standalone version works fine...

**Comment by jbaylies** [comment]
@peteone43 are you using an m1 mac? I wonder why it works for me but not for you.. you're using JTBA 2.1.16, right?

**Comment by berndkeul** [comment]
Yes I am on Mac mini M1, but I also use a Mac mini intel I7 (2018) with Monterey , Jamtaba 2.1.16 works sometimes and sometimes not, I reinstalled it but I could not figure out why it is inconsistent…Von meinem iPhone gesendetAm 17.02.2023 um 16:14 schrieb jbaylies ***@***.***>:﻿
@peteone43 are you using an m1 mac? I wonder why it works for me but not for you.. you're using JTBA 2.1.16, right?

**Comment by peteone43** [comment]
> @peteone43 are you using an m1 mac? I wonder why it works for me but not for you.. you're using JTBA 2.1.16, right?

Im actually using MacBook Pro i7 with Ventura 13.2 installed

**Comment by berndkeul** [comment]
I will try Ventura … 

Von meinem iPad gesendet

**Comment by one-agnostik** [comment]
The AU is probably not opening in Ableton (or other DAWs) because of the Mac security changes (SIP). Although it supposedly lets you override it in security and privacy, that doesn't actually work for plugins because they are run from within another program.. here is the fix...

You have to manually remove the "quarantine" status from the plugin which OSX assigns because it comes from an unauthorised developer.

To do this, open a terminal window located in your plugins > components and type the following:

xattr JamTaba.component

This is to check the attributes OSX has assigned the au. It should return "com.apple.quarantine" which means OSX is treating it as a suspect process.

Still within the  plugins > components directory, type the following to remove it:

sudo xattr -rd com.apple.quarantine JamTaba.component

Enter your password and that should tell osx it is safe an then the AU will work.

**Comment by one-agnostik** [comment]
An update to this: that fix seems to only be temporary. Au beachballing on ableton live 12 once again...! 

Have emailed the dev but no reply
